symplyphysics/laws/electricity/transmission_lines/transmission_matrix_of_pi_type_circuit.py

```diff
@@ -16,11 +17,11 @@
 ## Description
 ## The transmission parameters matrix is one of the ways to describe a microwave device. The ABCD-parameters of the device act as elements
 ## of this matrix. The matrix equation relates the input voltage and input current to the output voltage and output current.
-## The T-type circuit consists of the first impedance connected in series, the third impedance connected in parallel, and the
-## second impedance connected in series.
+## The π-type circuit consists of the first impedance connected in parallel, the third impedance connected in series, and the
```

Comment:
Somehow you've rewritten previous law. It should not happen.

Reply:
> Somehow you've rewritten previous law. It should not happen.

I don't understand, what's the problem?

Reply:
You've replaced T-type law with pi-type.

Reply:
> You've replaced T-type law with pi-type.

Yes. Because here we are talking about the impedances of the pi-type circuit, not the T-type.

Reply:
Yes, but t-type is also relevant. It should not be overwritten by new PR.